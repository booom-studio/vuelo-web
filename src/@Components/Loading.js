import React from 'react';
import PropTypes from 'prop-types';

import { withStyles } from 'material-ui/styles';
import { CircularProgress } from 'material-ui/Progress';

const styles = theme => ({
  container: {
    display: 'flex',
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: theme.palette.grey.A100
  }
});

const Loading = ({ classes }) => (
  <div className={classes.container}>
    <CircularProgress size={100} color="accent" thickness={1} />
  </div>
);

Loading.propTypes = {
  classes: PropTypes.object.isRequired
};

export default withStyles(styles)(Loading);
