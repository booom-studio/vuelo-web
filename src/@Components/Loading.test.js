import React from 'react';
import Loading from './Loading';

import renderer from 'react-test-renderer';

test('Loading', () => {
  const component = renderer.create(<Loading />);
  expect(component.toJSON()).toMatchSnapshot();
});
