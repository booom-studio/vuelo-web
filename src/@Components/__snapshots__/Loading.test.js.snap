// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading 1`] = `
<div
  className="Loading-container-1"
>
  <div
    className="MuiCircularProgress-root-2 MuiCircularProgress-accentColor-4"
    role="progressbar"
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svgIndeterminate-5"
      viewBox="0 0 50 50"
    >
      <circle
        className="MuiCircularProgress-circle-7 MuiCircularProgress-circleIndeterminate-8"
        cx={25}
        cy={25}
        fill="none"
        r={20}
        strokeWidth={1}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
